### PR TITLE
misc(release): delete lighthouse pristine folder to keep it fresh

### PR DESCRIPTION
**Pull request by paulirish**

### CODE REVIEW

File: lighthouse-core/scripts/release/prepare-pristine.sh [hunk 1]
```diff
@@ -9,10 +9,10 @@ set -euxo pipefail
 # Setup a pristine git environment
 cd ../
 
-if [[ ! -e lighthouse-pristine/ ]]; then
-  git clone git@github.com:GoogleChrome/lighthouse.git lighthouse-pristine
-fi
+# Delete the folder if it already exists. We need it super fresh
```
[the remark this answers is not in the document]
Comment (patrickhulce) [reply lighthouse-core/scripts/release/prepare-pristine.sh]: hm, that's so weird. alright then thanks for the explanation! :)

File: lighthouse-core/scripts/release/prepare-pristine.sh [hunk 2]
```diff
@@ -9,10 +9,10 @@ set -euxo pipefail
...
+rm -rf lighthouse-pristine
```
Comment (patrickhulce) [inline lighthouse-core/scripts/release/prepare-pristine.sh]: should be able to get rid of pretty much all the git commands after this then

File: lighthouse-core/scripts/release/prepare-pristine.sh [hunk 3]
```diff
@@ -9,10 +9,10 @@ set -euxo pipefail
...
 
+git clone git@github.com:GoogleChrome/lighthouse.git lighthouse-pristine
 cd lighthouse-pristine/
 
 if [[ -n "$(git status --porcelain)" ]]; then
```
Comment (patrickhulce) [inline lighthouse-core/scripts/release/prepare-pristine.sh]: should we move this warning about the `rm -rf` to prevent accidental deletion?

**Comment by brendankenny** [comment]
last time I ran the release scripts (maybe 5.0? I think I just did it manually for 5.2 :) I think I remember the zone.js issue being something different.

Was the error happening in the`yarn smoke` call or the `npm explore lighthouse -- npm run smoke` call in `test.sh`? I think the second one would be caused by the fact that [`static-server` serves `zone.js` out of `node_modules/`](https://github.com/GoogleChrome/lighthouse/blob/d0e5c58f690d067e6243b1531dffbdd6d86ec008/lighthouse-cli/test/fixtures/static-server.js#L50-L54), but since `zone.js` is in LH's devDeps it wouldn't be installed for the test of the module tarball.

You can see this in a random directory with just the version on npm if you do
```sh
npm init -y
npm install lighthouse
npm explore lighthouse -- npm run smoke pwa # no problem
npm explore lighthouse -- npm run smoke dbw # extra console error due to zone.js 404
```

**Comment by connorjclark** [comment]
@brendankenny figured it out :)

maybe we should have a --skip-test flag for smokes

**Comment by brendankenny** [comment]
no one this one after #9672?

**Pull request closed without merging**